### DISCUSSION TITLE
other models besides open ai

### DISCUSSION THREAD

**Discussion by ontop01970** [danielmiessler/Fabric#116] [General]
Daniel, very cool integration.  You hit the nail on the head, all these things are great, but if you cant integrate and implement them into your day to day workflow, they are of no use so thank you for that.  So my question is can we use some other LLM's to call on?  maybe one thats running locally(and of course open source)?

**Comment by danielmiessler** [comment]
Yes, we're looking to add local models very soon using locallm.
